www/_template/concepts/build-pipeline.md

```diff
@@ -3,38 +3,39 @@ layout: layouts/content.njk
 title: The Build Pipeline
 description: Snowpack Build creates a production-ready website with or without a bundler
 ---
+THe Snowpack build pipeline runs on the command `snowpack build`. Snowpack then creates a production-ready website, with or without a bundler.
 
-![build output example](/img/snowpack-build-example.png)
+This article will cover
+- What Snowpack does by default
+- When and how to use a bundler
 
-`snowpack build` - When you're ready to deploy your application, run the build command to generate a static production build of your site. Building is tightly integrated with your dev setup so that you are guaranteed to get a near-exact copy of the same code that you saw during development.
+Snowpack’s philosophy is **you should be able to use a bundler because you want to, and not because you need to.** Snowpack treats bundling as an optional production optimization, which means you're free to skip the extra complexity of bundling until you need it.
 
-### Bundle for Production
+Without a bundler, `snowpack build` generates a static production build of your site using the same unbundled approach as the `dev` command. For many projects, this is all you need.
 
-**You should be able to use a bundler because you want to, and not because you need to.** That was the original concept that Snowpack was designed to address. Snowpack treats bundling as an optional production optimization, which means you're free to skip over the extra complexity of bundling until you need it.
 
-By default, `snowpack build` will build your site using the same unbundled approach as the `dev` command. This is fine for most projects, but you also may still want to bundle for production. Legacy browser support, code minification, code-splitting, tree-shaking, dead code elimination, and other performance optimizations can all be handled in Snowpack via bundling.
+## Using a bundler
 
-Bundlers normally require dozens or even hundreds of lines of configuration, but with Snowpack it's just a one-line plugin with no config required. This is possible because Snowpack builds your application _before_ sending it to the bundler, so the bundler never sees your custom source code (JSX, TS, Svelte, Vue, etc.) and instead needs to worry only about building common HTML, CSS, and JS.
+Bundlers normally require dozens or even hundreds of lines of configuration, but adding a bundler to Snowpack is a one-line addition of a **bundler plugin** to `snowpack.config.js`. Snowpack then runs the bundler plugin automatically on `Snowpack build`.
```

Comment:
reader: "Why is it a one-line addition?" 

"This is possible because" sentence was removed which explained this.

Reply:
Oh, looks like this was just moved below to "Snowpack eliminates..."

Feels odd to split up the statement and the explanation with a code snippet between them. Can you reorganize to bring those two back together?